source/indexes/compound-index.txt

```diff
@@ -0,0 +1,124 @@
+.. _ruby-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+**Compound indexes** hold references to multiple
+fields within a collection's documents, improving query and sort
+performance. 
+
+When creating a compound index, you must specify the following details:
+
+- The fields on which to create the index
+
+- The sort order for each field (ascending or descending)
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the
+``sample_mflix`` database from the :atlas:`Atlas sample datasets
+</sample-data>`. To access this collection from your {+language+} 
+application, create a ``Mongo::Client`` object that connects to
+an Atlas cluster and assign the following values to your ``database`` 
+and ``collection``
+variables: 
+
+.. literalinclude:: /includes/indexes/single-field.rb
+   :start-after: start-sample-data
+   :end-before: end-sample-data
+   :language: ruby
+   :copyable:
+
+To learn how to create a free MongoDB Atlas cluster and
+load the sample datasets, see the :atlas:`Get Started with Atlas
+</getting-started>` guide.
+
+Create a Compound Index
+-----------------------
+
+Use the ``create_one`` method to create a compound index. The following 
+example creates an index in descending order on the ``runtime`` field and 
+in ascending order on the ``year`` field:
+
+.. literalinclude:: /includes/indexes/compound.rb
+   :start-after: start-index-compound
+   :end-before: end-index-compound
+   :language: ruby
+   :copyable:
+
+Verify Index Creation
+---------------------
+
```

Comment:
I feel like it's somewhat repetitive to include this section for every page for individual index types. I personally think it would suffice to include this code sample once in the top-level Indexes page.

Reply:
@mcmorisi Tried out linking to the Listing Indexes section on the overview page, but then users won't be able to see the output (as it doesn't seem to be the standard to show output on the overview page). The section is relatively small, so it doesn't make the page significantly longer. What do you think about keeping it?